File: WinFormsApp3/WinFormsApp3/Form1.vb
Public Class Form1
    'Var 3 Task3'
    Private Sub buttonShowGreeting_Click(sender As Object, e As EventArgs) Handles buttonShowGreeting.Click
        Dim name As String = textBoxName.Text

        If String.IsNullOrWhiteSpace(name) Then
            MessageBox.Show("Введите ваше имя, чтобы получить поздравление.")
            Return
        End If

        Dim greeting As String = $"Дорогий(а) {name}!" & vbCrLf & vbCrLf &
                                  "Щиро вітаю вас з днем народження!" & vbCrLf &
                                  "Бажаю вам здоров'я, щастя та успіхів!" & vbCrLf & vbCrLf &
                                  "З повагою," & vbCrLf &
                                  "Президент країни"

        labelGreeting.Text = greeting

        pictureBoxSignature.Visible = True
        PicSign.Visible = True

    End Sub
End Class

File: Lab1/Lab1_Task3/Lab1_Task3/Form1.cs
using System;
using System.Windows.Forms;

namespace Lab1_Task3
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            // Attach the event handler to the button
            buttonShowGreeting.Click += buttonShowGreeting_Click;
        }

        private void buttonShowGreeting_Click(object sender, EventArgs e)
        {
            string name = textBoxName.Text;

            if (string.IsNullOrWhiteSpace(name))
            {
                MessageBox.Show("Please enter your name to receive a greeting.");
                return;
            }

            string greeting = $"Dear {name}!" + Environment.NewLine + Environment.NewLine +
                              "Happy Birthday!" + Environment.NewLine +
                              "Wishing you health, happiness, and success!" + Environment.NewLine + Environment.NewLine +
                              "Best regards," + Environment.NewLine +
                              "The President of Ukraine";

            labelGreeting.Text = greeting;
        }
    }
}
